src/tile/tile.scss

```diff
@@ -196,13 +191,13 @@
         color: t(iui-text-color-muted);
       }
 
-      > svg {
+      .iui-metadata-icon {
         @include iui-icons-default;
         margin-right: $iui-s;
         flex-shrink: 0;
       }
 
-      > *:not(svg) {
+      > * {
```

Comment:
why not svg removed?

Reply:
These rules don't influence svgs, so no need